website/docs/d/tg_gateway.html.markdown

```diff
@@ -45,20 +45,21 @@ In addition to the argument reference list, you can access the following attribu
   Nested scheme for `connections`:
 	- `created_at` - (String) The date and time the connection is created.
 	- `id` - (String) The unique identifier for the transit gateway connection to network either `VPC`,  `classic`.
-    - `base_connection_id` - The ID of a network_type `classic` connection a tunnel is configured over.  This field only applies to network type `gre_tunnel` connections.
+  - `base_connection_id` - The ID of a network_type `classic` connection a tunnel is configured over.  This field applies to network type `gre_tunnel` or `unbound_gre_tunnel` connections.
+  - `base_network_type` - The type of network the unbound gre tunnel is targeting. This field is required for network type `unbound_gre_tunnel`.
```

Comment:
The indentation is missed out
![Uploading Screenshot 2022-12-12 at 12.30.17 PM.png…]()

Reply:
The indentations have been fixed.